fitbenchmarking/fitbenchmarking
# Removing redundent fitting/misc.py

#### Description of Work

Fixes #371 

#### Testing Instructions

1. Check fitting/misc.py not imported

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
